recod_erpnext_design/hooks.py

```diff
@@ -17,7 +17,7 @@
 
 # Includes in <head>
 # ------------------
-# fixtures = [""]
+fixtures = ["Print Format"]
```

Comment:
Shouldn't there also be `Terms and Conditions` and `Website Theme` in this list ??